website/blog/2022-12-09-m365-sandbox-setup.md

```diff
@@ -0,0 +1,83 @@
+---
+slug: m365-sandbox-setup
+title: "Setting up your own Microsoft 365 sandbox"
+authors: nica
+tags: [corso, microsoft 365]
+date: 2022-12-09
+image: ./images/ar_sandbox.jpg
+---
+When trying to do this myself I found some of the documentation out of date,
+so this is a quick guide to setting up a sandbox domain for Microsoft 365 development.
+
+## Why would you want to do this?
+
+I personally needed this for recording a screencast of the process to install the free CLI tool Corso.
+While recording myself going through permission settings in M365,
+I didn’t want to unintentionally reveal PII for my team members or other sensitive data.
+
+Another reason if you’re looking to level up both your skills,
+you may want to experiment with being the admin in your own space,
+rather than bumping into other users on a production domain. It’s also a good step to experiment with [owning your own identity](https://corsobackup.io/blog/your-own-backups/).
```

Comment:
```suggestion
rather than impacting other users on a production domain. It’s also a good step to experiment with [owning your own identity](https://corsobackup.io/blog/your-own-backups/).
```